deicode/_rpca_defaults.py

```diff
@@ -4,12 +4,24 @@
 DEFAULT_RANK = 3
 DEFAULT_MSC = 500
 DEFAULT_MFC = 10
+DEFAULT_MFF = 0
 DEFAULT_ITERATIONS = 5
 
-DESC_RANK = ("The underlying low-rank structure (suggested: 1 < rank < 10)"
-             " [minimum 2]")
-DESC_MSC = "Minimum sum cutoff of sample across all features"
-DESC_MFC = "Minimum sum cutoff of features across all samples"
+DESC_RANK = ("The underlying low-rank structure."
+             " The input can be an integer "
+             "(suggested: 1 < rank < 10) [minimum 2]."
+             " Note: as the rank increases the runtime"
+             " will increase dramatically.")
+DESC_MSC = ("Minimum sum cutoff of sample across all features"
```

Comment:
```suggestion
DESC_MSC = ("Minimum sum cutoff of sample across all features. "
```
The space will look weird here otherwise.

Reply:
Done, thanks!